sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/Models/CommonDataServiceForAppsLinkedService.cs

```diff
@@ -172,11 +169,10 @@ public CommonDataServiceForAppsLinkedService(string deploymentType, string authe
         /// Service for Apps server. 'Office365' for online scenario, 'Ifd' for
         /// on-premises with Ifd scenario. 'AADServicePrincipal' for
         /// Server-To-Server authentication in online scenario. Type: string
-        /// (or Expression with resultType string). Possible values include:
-        /// 'Office365', 'Ifd', 'AADServicePrincipal'
+        /// (or Expression with resultType string).
         /// </summary>
         [JsonProperty(PropertyName = "typeProperties.authenticationType")]
-        public string AuthenticationType { get; set; }
+        public object AuthenticationType { get; set; }
```

Comment:
Why are these fields being changed from string to object? The description still says Type: string

Reply:
It is fix swagger S360 Linting issue, You can find the link in this https://portal.azure-devex-tools.com/amekpis/linting/detail?errorId=4394BD41-F5A8-45DC-B526-4F915203EF9D

for more, in DataFactory we support these fields both string and expression with resultType string. So we made these field into object.